src/main/resources/META-INF/resources/frontend/idle-notification.js

```diff
@@ -344,8 +350,10 @@ class IdleNotification extends ThemableMixin(PolymerElement) {
   /** @protected */
   disconnectedCallback() {
     super.disconnectedCallback();
-    this.removeEventListener('load', this._handleLoad);
+    this.removeEventListener('load', this._handleLoadListener);
     this._clearTimeoutObject();
+    // to not trigger _handleLoad after disconnecting the component
+    this._displayProcessStarted = true;
```

Comment:
@tarekoraby this part is my least favourite. I would rather get rid of the custom XMLHttpRequest.prototype.open function instead but don't know how to do that.

Reply:
I think it's fine. It serves the purpose.